app/controllers/api/auth/passwords_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  module Auth
+    class PasswordsController < DeviseTokenAuth::PasswordsController
+      skip_before_action :verify_authenticity_token
+      before_action :configure_permitted_parameters, if: :devise_controller?
+
+      def edit        
```

Comment:
What about a test for this? 🤔 

`GET api/v1/users/passwords/edit`

Reply:
I was thinking in leaving the tests with rspec for another pull entirely !